controllers/storagecluster/noobaa_system_reconciler.go

```diff
@@ -119,9 +119,13 @@ func (r *StorageClusterReconciler) setNooBaaDesiredState(nb *nbv1.NooBaa, sc *oc
 
 	nb.Spec.CoreResources = &coreResources
 	nb.Spec.DBResources = &dbResources
-	placement := getPlacement(sc, "noobaa-core")
-	nb.Spec.Tolerations = placement.Tolerations
-	nb.Spec.Affinity = &corev1.Affinity{NodeAffinity: placement.NodeAffinity}
+	isStandalone := sc.Spec.MultiCloudGateway != nil &&
```

Comment:
r.IsNoobaaStandalone already has this information (line 114 in this file)

Reply:
...oops! you're right!